ibtk/include/ibtk/FEDataManager.h

```diff
@@ -294,9 +294,25 @@ class SubdomainToLevelTranslation
         return it->second;
     }
 
+    /*!
+     * Return whether or not there are any elements on a given level.
+     */
+    bool levelHasElements(const int level_number) const
```

Comment:
👍

Reply:
Unless there is a bug in the implementation the check that uses this should prevent us from calculating the global set of bounding boxes any more times than we currently do on `master`.